rubocop/rubocop_rspec.yml

```
@@ -0,0 +1,30 @@
+require:
+  - rubocop-rspec
+
+# Request and system specs by definition do not test specific classes/modules,
+# so they are excluded from following the `describe ClassOrModule` rule.
+RSpec/DescribeClass:
+  Exclude:
+    - "spec/requests/**/*"
+    - "spec/system/**/*"
+
+# Prefer short examples but allow some flexibility (up to 12 lines). System
+# specs are excluded from this cop because they often involve many sequential
+# steps, and forcing these to be shorter can make them harder to follow.
+RSpec/ExampleLength:
+  Max: 12 # Default is 5
+  Exclude:
+    - "spec/system/**/*"
+
+# If an `it` block does not have a description (i.e. the example relies on
+# RSpec magic to infer the description), then prefer it use the `is_expected`
+# syntax instead of `expect(subject)`, like this:
+# it { is_expected.to be_truthy }
+RSpec/ImplicitSubject:
+  EnforcedStyle: single_statement_only
+
+# Ideal BDD is to have one `expect` per example (so that each specification can
+# have corresponding documentation in the form of an `it` block), but we allow
+# some flexibility (up to 3).
+RSpec/MultipleExpectations:
+  Max: 3 # Default is 1
```

Because system tests are often long, the also often have many expectations.
Perhaps:
```suggestion
  Max: 3 # Default is 1
  Exclude:
    - "spec/requests/**/*"
    - "spec/system/**/*"

```

---

Makes sense 👍

---

Would you be OK with excluding only `system` and not `requests`? In my experience request specs are used for APIs and those are easier to write bite-sized. Whereas system specs are long capybara flows.

---

Hmmmm, I put capybara "system" specs under `spec/features/`.

---

leaving out request specs seems fine.  so `spec/features/**` and `spec/system/**` ?

---

i see that's done!